flink-runtime/src/test/java/org/apache/flink/runtime/io/network/partition/ResultPartitionTest.java

```diff
@@ -67,7 +67,7 @@ public void testSendScheduleOrUpdateConsumersMessage() throws Exception {
 			partition.addBufferConsumer(createFilledBufferConsumer(BufferBuilderTestUtils.BUFFER_SIZE), 0);
 			verify(notifier, times(1))
 				.notifyPartitionConsumable(
-					eq(partition.getJobId()),
+					any(JobID.class),
```

Comment:
I guess we could still keep `@VisibleForTesting ResultPartition.getJobId()`?

Reply:
Yes, it can be package private method.